.circleci/common.sh

```diff
@@ -123,7 +102,7 @@ function install_deps_pytorch_xla() {
 function build_torch_xla() {
   XLA_DIR=$1
   pushd "$XLA_DIR"
-  python setup.py install
+  python setup.py install --user
```

Comment:
I wonder why we start to install in the users' directory now

Reply:
I found `--user` is needed when jenkins is a normal user and python env is system python. Since we keep using conda now, the `--user` isn't needed anymore because the python env is setup by jenkins